### ISSUE TITLE
Play function should work when quitting Clip Tabbing Mode

### ISSUE THREAD

**Issue by chinakov** [audacity/audacity#8764]
### Description

- If the User has engaged the Clip Tabbing mode, he can use the spacebar during this clip tabbing mode to select and deselect clips. However, once the User has quit the clip tabbing mode via moving focus away from the clips view, the User is still unable to use spacebar to initiate playback. The User should be able to use the 'spacebar' to initiate playback if the clips view is not in focus. 
- When the User has realtime Effects dialog window in focus, the User should be able to press 'spacebar' to initiate or stop playback, and use the 'Enter' button to enable or disable the effect. Currently, the 'spacebar' enables or disables the effect. Enter does nothing currently.

**Comment by chinakov** [comment]
This was fixed via https://github.com/audacity/audacity/issues/8751, closing.
